word2vec/sketch.js

```diff
@@ -7,34 +7,71 @@ let wordVecs;
 function setup() {
   createCanvas(100, 100);
 
+  let loadHide = select("#loadHide");
+  loadHide.hide();
+
   let wordInput = select('#word');
   let button = select('#submit');
-  wordInput.hide();
-  button.hide();
   let results = select('#results');
 
+  let between1 = select("#between1");
+  let between2 = select("#between2");
+  let button2 = select("#submit2");
+  let results2 = select("#results2");
+
+  let isto1 = select("#isto1");
+  let isto2 = select("#isto2");
+  let isto3 = select("#isto3");
+  let button3 = select("#submit3");
+  let results3 = select("#results3");
+
   loadJSON('data/wordvecs10000.json', (data) => {
     wordVecs = data.vectors;
     console.log('loaded');
-    wordInput.show();
-    button.show();
+    loadHide.show();
     noLoop();
     noCanvas();
   });
 
-
   button.mousePressed(() => {
-    word = wordInput.value();
-    let nearest = Word2Vec.nearest(word, 10);
-    if (!nearest) {
-      results.html('no word vector found');
+    let word = wordInput.value();
+    results.html(findNearest(word, 10));
+  });
+
+  button2.mousePressed(() => {
```

Comment:
maybe rename buttons to be descriptive rather than numbered? `buttonNearest`, `buttonAdd`, etc.?

Reply:
I definitely thought about that while making this. Will take a look at it.